parkbenchcommon/broadcastconsumer.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2017-2018 Joel Allen Luellwitz and Andrew Klapp
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Provides the consuming component of a filesystem-based IPC mechanism."""
+
+import datetime
+import os
+import logging
+import time
+import tmpfs
+
+__all__ = ['BroadcastCheckError', 'BroadcastConsumer']
+
+BROADCAST_PATH = '/var/spool/'
```

Comment:
I'd call this SPOOL_DIR or SPOOL_PATH. (See what I do in other applications.) Also, this constant is inconsistent with defining the variable locally in Broadcaster. Because this variable is unlikely to be referenced twice and is unlikely to change, I would declare it right before you use it.

Reply:
Actually, I changed my mind on this. I'd define it here as it needs to match the Broadcaster class. You should do the same in the Broadcaster class.